app/controllers/send_application_controller.rb

```diff
@@ -19,13 +19,16 @@ def update
 
   def send_email
     ApplicationMailer.snap_application_notification(
-      file_name: new_file_name,
+      file_name: "tmp/#{new_file_name}",
       recipient_email: current_snap_application.email,
     ).deliver
   end
 
   def create_dhs1171_pdf
-    Dhs1171Pdf.new(current_snap_application).save(new_file_name)
+    Dhs1171Pdf.new(
+      snap_application: current_snap_application,
+      output_filename: new_file_name,
```

Comment:
Yeah it is a good suggestion but is breaking the interim file thing where I append `working_` or equivalent to the filename. It either mangles the path or the extension. I'm going to leave this for now and revisit later.